requirements/requirements.txt

```diff
@@ -0,0 +1,9 @@
+sphinx==3.5.4
+sphinx-rtd-theme==1.0.0
+sphinxcontrib-bibtex==2.4.1
+matplotlib==3.3.4
+nbsphinx==0.8.7
+rst2pdf==0.98
+pytest==6.2.4
+pytest-cov==2.12.1
+pytest-mock==3.6.1
```

Comment:
You made a mix between requirements and dev-requirements: ingenialink, ingenialogger and ifaddr should be requirements, and the rest to dev.